usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/whiteboardReset.js

```diff
@@ -0,0 +1,66 @@
+//
+//  whiteboardReset.js
+//
+//  created by Rebecca Stankus on 03/28/19
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+(function() {
+
+    var _this;
+
+    var RESET_SOUND = SoundCache.getSound(Script.resolvePath('../resources/sounds/resetWhiteboard.mp3'));
+    var RESET_SOUND_VOLUME = 0.05;
+    var SEARCH_RADIUS_M = 6;
+
+    var injector;
+
+    var WhiteboardReset = function() {
+        _this = this;
+    };
+
+    WhiteboardReset.prototype = {
+
+        /* ON PRELOAD: Save a reference to this */
+        preload: function(entityID) {
+            _this.entityID = entityID;
+        },
+
+        /* PLAY A SOUND: Plays the specified sound at the position of the user's Avatar using the volume and playback 
+        mode requested. */
+        playSound: function(sound, volume, position, localOnly, loop){
+            if (sound.downloaded) {
+                if (injector) {
+                    injector.stop();
+                    injector = null;
+                }
+                injector = Audio.playSound(sound, {
+                    position: position,
+                    volume: volume,
+                    localOnly: localOnly,
+                    loop: loop
+                });
+            }
+        },
+
+        /* When clicked or triggered, find all nearby whiteboard lines and delete them */
+        mousePressOnEntity: function( entityID, event ) {
```

Comment:
this only works for lines that _i've_ drawn, right? it doesn't clear the whiteboard of everyone's lines, right?

Reply:
it clears all lines on the whiteboard to enable someone to come in and erase it before a meeting. For lines drawn in air, they are avatar entities and will persist for 60 seconds

Reply:
how does this clear everybody's line? if you pressed the reset button, i don't see how this code could erase my lines

Reply:
lines on the board are not avatar entities

Reply:
ahhhh! got it. okay 👍👍👍